README.md

```diff
@@ -1,2 +1,478 @@
+    `secret_key_base` section.
+
+  4. config/unicorn.rb
+
+    You can mostly use the sample file as-is, unless you use a different Linux
+    username.
+
+  5. config/google_omniauth.yml
+
+    Zartan uses Google oauth to authenticate users to the admin UI.  This file
+    allows google to perform this authentication.
+    This file needs modification in both development and production.  To create
+    `google_client_id` and `google_client_secret` fields, create [Google oauth
+    2.0 credentials]
+    (https://developers.google.com/youtube/registering_an_application).
+    Set the redirect uri to http://HOSTNAME/auth/google_oauth2/callback.
+    You can set the HOSTNAME to localhost for development and use the same
+    key/secret among your developers so long as they all use the same ports.
+
+  6. config/resque_schedule.yml
+
+    The sample file can be used as-is, but it can be modified if any schedules
+    need to be tweaked.
+
+  7. config/auth.yml
+
+    The `allowed_domains` setting must
+    be filled in with a domain which is hosted by google.com.  More nuanced
+    authentication settings could go here if they're implemented.
+
+8. Create an ssh key for the server to pull the source code from github
+
+  ```
+  ssh-keygen
+  cat ~/.ssh/id_rsa.pub
+  # Copy the key and add it to your github fork of the Zartan repo as a deploy key
+  ```
+9. Create config/deploy/production.rb in your dev environment.
+
+  Like the other config files, this has a `.sample` file.  However, this file
+  does not get uploaded to the production server.
+
+10. Initial Deploy. In the dev environment:
+
+  ```
+  # may require "bundle exec" depending on your setup
+  cap production deploy --trace
+  ```
+  This step succeeds if the final step complains about monit not being installed.
+  We couldn't install monit before now because it depends on files created by
+  the initial deploy.  For now, launch the various services manually and install
+  monit once everything is working.
+
+11. First Launch.
+
+  On the target machine, as "superuser":
+  Add executable init scripts for unicorn, resque_scheduler and resque_pool to
+  /etc/init.d. Sample files are located in
+  [config/samples/init.d](config/samples/init.d).  These files
+  will work as-is if your application's linux username is `zartan` and rvm was
+  installed by the `zartan` user (not the superuser).  Change the `USER` in each
+  of the files if the `USER` is not zartan, and change the final `PATH` directory
+  to `/usr/local/rvm/wrappers/ruby-2.2.0@zartan` if rvm was installed by the
+  superuser.
+  ```
+  sudo service zartan_unicorn start
+  sudo service zartan_resque_pool start
+  # Verify that unicorn and resque processes are running
+  ps -ef | egrep 'unicorn|resque'
+  ```
+
+12. Set up nginx.
+
+  Copy the config file in [config/samples/nginx/zartan](config/samples/nginx/zartan)
+  to `/etc/nginx/sites-enabled/zartan` on the target machine.  This file should
```

Comment:
To fit with convention, the file should actually be called `/etc/nginx/sites-available/zartan`, and that should then be symlinked into `/etc/nginx/sites-enabled/`